src/components/Analysis/Modes/ModesList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { isQueryLoading, useQuery } from 'cozy-client'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+
+import { buildGeoJSONQuery } from 'src/queries/queries'
+import LoadedModesList from 'src/components/Analysis/Modes/LoadedModesList'
+
+const ModesList = () => {
+  const geoJsonQuery = buildGeoJSONQuery()
```

Comment:
This query is quite dangerous: I think it would worth adding a warning saying what it does - query ALL the docs at once, while they are quite heavy - and reflecting it in the query name, e.g. `buildGeoJSONQueryNoLimit`.
It might also worth it to add a TODO mentioning what we discussed about performances  ;) I'm still unsure how it would react when there are thousands of trips to load/compute